google/cloud/pubsub/internal/defaults.cc

```diff
@@ -105,6 +106,13 @@ Options DefaultPublisherOptionsOnly(Options opts) {
   if (!opts.has<pubsub::CompressionAlgorithmOption>()) {
     opts.set<pubsub::CompressionAlgorithmOption>(GRPC_COMPRESS_DEFLATE);
   }
+  auto e = internal::GetEnv("OTEL_SPAN_LINK_COUNT_LIMIT");
+  size_t link_count;
+  if (e && absl::SimpleAtoi(e.value(), &link_count)) {
+    opts.set<pubsub::MaxOtelLinkCountOption>(link_count);
+  } else if (!opts.has<pubsub::MaxOtelLinkCountOption>()) {
+    opts.set<pubsub::MaxOtelLinkCountOption>(128);
```

Comment:
Is there some way we can discover Open Telemetry's default, rather than encoding a fixed number?

Reply:
I don't see a way to access it.